tests/testthat/test-modules.R

```diff
@@ -541,43 +415,43 @@ get_srv_and_ui <- function() {
   ))
 }
 
-testthat::test_that("is_reporter_used throws error if object is not teal_module or teal_modules", {
-  testthat::expect_error(is_reporter_used(5), "is_reporter_used function not implemented for this object")
-  testthat::expect_error(is_reporter_used(list()), "is_reporter_used function not implemented for this object")
+testthat::test_that("is_args_used throws error if object is not teal_module or teal_modules", {
+  testthat::expect_error(is_args_used(5, "reporter"), "is_args_used function not implemented for this object")
+  testthat::expect_error(is_args_used(list(), "reporter"), "is_args_used function not implemented for this object")
 })
 
-testthat::test_that("is_reporter_used returns true if teal_module has reporter in server function args", {
-  testthat::expect_true(is_reporter_used(module(server = function(id, datasets, reporter) NULL)))
+testthat::test_that("is_args_used returns true if teal_module has reporter in server function args", {
+  testthat::expect_true(is_args_used(module(server = function(id, datasets, reporter) NULL), "reporter"))
 })
 
-testthat::test_that("is_reporter_used returns false if teal_module does not have reporter in server function args", {
-  testthat::expect_false(is_reporter_used(module()))
+testthat::test_that("is_args_used returns false if teal_module does not have reporter in server function args", {
```

Comment:
These test names are a bit odd now probably need `is_args_used with args = reporter` (and I guess we can use arbitrary args not just reporter now).

We should probably also test is_args_used.function (does not need to be a ui and server function of course)

Reply:
I changed `is_args_used` (plural) to `is_arg_used` (single) because `arg` should probably be a single character instead of multiple.  
With multiple there is dilemma:
- should all together be present in any of the function or any.

Because we don't need multiple version now I have simplified a little.